crates/packager/src/config/mod.rs

```diff
@@ -488,6 +509,84 @@ impl AppImageConfig {
     }
 }
 
+/// The Linux pacman configuration.
+#[derive(Clone, Debug, Default, Deserialize, Serialize)]
+#[cfg_attr(feature = "schema", derive(schemars::JsonSchema))]
+#[serde(rename_all = "camelCase", deny_unknown_fields)]
+#[non_exhaustive]
+pub struct PacmanConfig {
+    // Arch Linux Specific settings.
+    /// List of Pacman dependencies.
+    pub depends: Option<Vec<String>>,
+    /// Additional packages that are provided by this app.
+    pub provides: Option<Vec<String>>,
+    /// Packages that conflict with the app.
+    pub conflicts: Option<Vec<String>>,
+    /// Only use if this app replaces some obsolete packages
+    pub replaces: Option<Vec<String>>,
+    /// Source of the package to be stored at PKGBUILD.
+    /// PKGBUILD is a bash script, so version can be referred as ${pkgver}
+    pub source: Option<Vec<String>>,
```

Comment:
Can we give a bit of more information about each option and add some links to the official docs?

Reply:
Sure